regression/tests/studio/test_studio_grading.py

```diff
@@ -4,6 +4,8 @@
 from bok_choy.web_app_test import WebAppTest
 from regression.pages.studio.login_studio import StudioLogin
 from regression.pages.studio.grading_studio import GradingPageExtended
+from regression.pages.studio.course_outline_page \
```

Comment:
nit: Another instance where you ought to be using the parens instead of the slash.